android/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,3 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="com.dooboolab.fluttersound">
+   package="com.dooboolab.fluttersound">
```

Comment:
2 spaces to 3 spaces?

Reply:
3 spaces is not good. I am going to fix that